### ISSUE TITLE
Values, Language, and Metrics

### ISSUE THREAD

**Issue by jorydotcom** [w3c/charter-html#12]
Thanks for inviting me to take a look at the doc, @marcoscaceres - I’m in awe of all that the W3C and its myriad groups accomplish for the web, and thank you all very much for all that you do! 

My assumption is that the Big Goal of this document is to improve participation in W3C systems. To be clear, I think the idea is great and seek only to help improve it. Having read through this doc a few times, I think there are a few themes that warrant consideration among your team(s):
1. System Values
2. Language
3. Metric Consensus

I’ve organized my thoughts around these themes, and hope that organization effectively presents the lens through which I’ve provided specific feedback. 
### System Values

This doc outlines two systems with different primary values: a Community Group system with the primary value of ideation (generating new ideas/innovation), and a Working Group system with the primary value of focus (managing incremental change). Ideally, these two systems are designed such that their values complement each other with healthy tension in a positive feedback loop, resulting in both groups being more productive. Unfortunately, this difference can also result in too much tension between values, and neither system will operate to its full capacity. 

To mitigate this, I suggest looking at the components of the process and identifying the primary value each represents. Perhaps you can have one person read through the doc for each system/value, and have them point out language or areas that might be in tension with the value they read the document with. Here are some things that caught me as potential areas of values tension:
- _“transitioning of ideas towards the standardization track”_ - raises the question, “What is a standardizable idea?” Being made to focus only on “standardizable” ideas might really frustrate innovative thinkers. Suggest creating separate guidelines for the kinds of ideas you’re looking for, and also for what things you’re looking for in a standardizable idea.
- _“proper continuity in participation”_ - Inclusivity is a value I picked up on in this doc. “Continuity in participation” may be read to work against that value, as your CG participators may not be members of W3C organizations and able to participate across the org. Suggest rephrasing this to be more specific (“x% participation across groups,” “x number of active CG members also following WG work” etc.). 
- _“A major objective of this new plan is to encourage participation by those who can best drive innovation and interoperability for the web platform.”_ - Ideators aren’t often the best Focusers, and this sentence may deter participation from people who can’t do both well. I think this can be rephrased but am drawing a blank at present. 

To reiterate, a certain amount of tension is really good and healthy - I’m only suggesting we take a look at where the tension might arise from a values perspective, as this is a good way to identify how we can resolve tension when/if it becomes too much.
### Language

There are a few terms and ideas that will have different meanings in each system:
-  _“mature enough to justify formal standardization”_ - recommend guidelines for what qualifies something as “mature enough” or else getting more specific (“There is a waiting period of x days,” etc.) 
- _“interoperability”_ - “interoperability in the field” and “actual interoperability” are confusing phrases, and in my mind, somewhat disruptive to the system you’re creating. As they are referred to in the document, they seem to set up an implicit hierarchy wherein the WG always supersedes the CG, which reinforces older about how spec development is done (see #7). This phrasing could be counter-productive to your inclusivity goal and certainly would make it harder to recruit and retain CG members. Further, it seems like the WG is in charge of actual interoperability, but is being directed to prioritize based on field interoperability, which could also create tension for members of both groups.
### Metric Consensus

The Document is a bit of an attempt to help people think more positively of standardization work as a team sport, which is awesome. One way to do that is to make sure goals for the “teams” are clear - some of that will come from group leadership, but this will be more effective if it is peer driven. The W3C should consider what metrics it might use to evaluate leadership and team success for each group. A few thoughts:
- Name/Suggest quantifiable goals - be it a certain number of requests to transition, time from idea to Rec, number of participants, etc
- Specify leadership roles even further: What does effective work look like for this person? Do they share complementary values with the system they are leading? (related to #11 )
- What will you use to measure whether information/ideas are effectively being exchanged across working groups and the CG?
### Final thoughts

One last thought that didn’t fit within a theme is on WG membership participating in the CG. It can be really challenging for people to set aside a role in one group to go play in another, very similar group. I agree that you want to encourage cross-collaboration/participation, but would encourage monitoring cross-group participation to ensure you are not accidentally undermining the effectiveness of either group. If it is a real concern, you may want to consider a target like, “x% of CG membership should be from the general public.”

I hope this perspective proves helpful to your work - Thanks for reading!

**Comment by w3cmemes** [comment]
HI, Key #w3c stakeholder here and new to github but just wading in so to speak to shake a tamborine in big +1 of support for the witty & entertaining freestyle musings above in honor of HTML.

AMong many things said by the writer of this one can probably best like how it talks about the positive feedback loop as it were healthy and actually postive to have such rather than in the normal universal way its otherwise used for meaning a thing thats eg osciallating out of control such as the Tacoma Narrows Bridge etc. NO Tacoma Narrows Bridge here though!

Generally one can appreciate the thought put into this in contrast to what some other writers might do instead as far as just say offering up a some really obvious advise that anybody 12 years or older who's spent 10 minutes reading the candidate recommendation charter and only-half thinking about it could come up with on there own.

Another part to step back and be admired is how this work avoids trying to do something like dress up several mundane ideas in a few fancy words it innovates with the words in teh same spirit as HTML itself innovates with other kinds of things. One example is it has the word "tension" in several places to mirror the tension you feel when try to read through it. Another example is it puts foward words like "ideation" and "Ideators" which yeah they may make you giggle at first but then some of the profundity of those  creeps into you later, along with the tension. 

SO in other words it's a good match and in summary there is clearly no BS here and its not only somebody strolling in to just talk out of there hat.

**Comment by AFBarstow** [comment]
Thanks very much @jorydotcom for your comments! Please don't let the drivel from @w3cmemes discourage you from continuing to participate in this project and other projects trying to help move the Web forward.

**Comment by jorydotcom** [comment]
:smile:  @AFBarstow - I can choose either to read @w3cmemes' comment as bad sarcasm or support, so I choose the latter. I aim only to be helpful, and defer to you, @marcoscaceres, @stevefaulkner and others who seem to be interested in this perspective as to whether that is the case! If it is not, kindly let me know and I will get on with doing whatever else it is I'm supposed to be doing today :smile:

**Comment by marcoscaceres** [comment]
I've reported @w3cmemes to github (are you back, Mr Lastweek? cause we've missed you). @plehegar, maybe delete the unhelpful comment. That's not associated with the actual w3c memes account on Twitter. Apologies.

**Comment by plehegar** [comment]
I deleted the comment from @w3cmemes.
